content/applications/finance/accounting/customer_invoices/electronic_invoicing.rst

```diff
@@ -51,6 +51,8 @@ Odoo supports, among others, the following formats.
      - For Singaporean companies
    * - A-NZ BIS Billing 3.0
      - For Australian/New Zealand companies
+   * - Peppol
+     - For :ref:`Peppol<e-invoicing/peppol>`
```

Comment:
I don't get it, Peppol is not a format ...? Or what's the difference with the Peppol BIS3 ?